src/api_client.rs

```diff
@@ -395,8 +396,19 @@ impl Client {
         unimplemented!("")
     }
 
-    pub async fn get_health() -> Result<HealthStatus, Error> {
-        unimplemented!("")
+    pub async fn get_health(&self) -> Result<HealthStatus, Error> {
+        let path = "eth/v1/node/health";
+        let target = self.endpoint.join(path)?;
+        let request = self.http.get(target);
+        let response = request.send().await?;
+        let status: u16 = StatusCode::as_u16(&response.status());
+        let result = match status {
+            200 => HealthStatus::Ready,
+            206 => HealthStatus::Syncing,
+            503 => HealthStatus::NotInitialized,
```

Comment:
can match on the hard-coded constants defined in `http` like

```suggestion
            http::OK => HealthStatus::Ready,
```

and skip the conversion to a `u16`

Reply:
ah ok, got it